todo_app/templates/index.html

```diff
@@ -8,11 +8,44 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-md-8">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <div class="card">
+        <ul class="list-group list-group-flush">
+          {% for item in items %}
+            <li class="list-group-item d-flex justify-content-between align-items-center">
+              {% if item.status == 'Completed' %}
+                <strike>{{ item.title }}</strike>
+              {% else %}
+                {{ item.title }}
+              {% endif %}
+              <div class="btn-group">
+                {% if item.status != 'Completed' %}
+                  <form method="post" action="/items/{{ item.id }}">
```

Comment:
You can also use Flask's url_for function in these templates. 

E.g. `action="{{ url_for('update_item', id=item.id) }}"`